src/lib/index.js

```diff
@@ -98,29 +99,46 @@ lib.pauseEvent = function(e) {
  * ------------------------------------------
  */
 
-// set VERBOSE to true to get a lot more logging and tracing
-lib.VERBOSE = false;
+/* eslint-disable no-console */
+lib.log = function() {
```

Comment:
can you put the new `lib.log`, `lib.warn` and `lib.error` in a separate file?

I'd vote for `src/lib/loggers.js`.